echo/.vscode/settings.json

```
@@ -53,6 +53,11 @@
   ],
   "files.eol": "\n",
   "debug.internalConsoleOptions": "neverOpen",
+  "python.testing.unittestEnabled": false,
```

Biome (1.9.4)

[error] The key python.testing.unittestEnabled was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)